spec/features/range_date_time_picker_filter_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+describe "Range DateTime Picker Filter", type: :feature do
+  before do
+    register_page(Invoice) do
+      filter :updated_at, as: :range_date_time_picker      
+    end
+
+    5.times { |i| Invoice.create() }
```

Comment:
Unused block argument - i. You can omit the argument if you don't care about it.<br>Do not use parentheses for method calls with no arguments.